### PR TITLE
Event featured chapters

### PR DESCRIPTION
**Pull request by BasVoorhoede**

### CODE REVIEW

File: components/event-block/EventBlockRelatedStories.vue [hunk 1]
```diff
@@ -0,0 +1,190 @@
+<template functional>
+  <div>
+    <img
+      v-if="props.showWaveMarker"
+      src="/event-title-wave.svg"
+      width="111"
+      height="35"
+      alt=""
+      class="event-block-related-stories__icon"
+    >
```
Comment (Siilwyn) [inline components/event-block/EventBlockRelatedStories.vue:10]: Do we also show this wave here? Can't find it in the design.

File: components/event-block/EventBlockRelatedStories.vue [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+        <nuxt-link
+          :to="`/narratives/${linkedChapter.book.slug}/${linkedChapter.chapter.slug}`"
+          class="event-block-related-stories__link"
+        >
+          <div class="event-block-related-stories__chapter-title">
+            {{ linkedChapter.chapter.title }}
+          </div>
+
+          <img
+            :src="linkedChapter.chapter.cover.url"
```
Comment (Siilwyn) [inline components/event-block/EventBlockRelatedStories.vue:41]: Think this is missing imgix compress parameters?

File: components/event-block/EventBlockRelatedStories.vue [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  }
+
+  .event-block-related-stories__link:before {
+    content: '';
+    display: block;
+    position: absolute;
+    z-index: 1;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(180deg, rgba(0, 0, 0, 0) 68%, rgba(0, 0, 0, 0.7));
```
Comment (Siilwyn) [inline components/event-block/EventBlockRelatedStories.vue:164]: Might want to increase the height and darkness of this gradient? I still find the text hard to read :thought_balloon: and/or add a text-shadow to the text.

File: pages/_language/events/_event/index.vue [hunk 4]
```diff
@@ -164,6 +173,27 @@
...
                   }
                 }
+                ... on RelatedStoriesBlockRecord {
+                  _modelApiKey
+                  id
+                  title
+                  subtitle
+                  titleColor
+                  showWaveMarker
+                  linkedChapters {
```
Comment (Siilwyn) [inline pages/_language/events/_event/index.vue:183]: Nice model name! Couldn't think of one when we discussed this

File: components/event-block/EventBlockRelatedStories.vue (hunk 1 above)
Comment (BasVoorhoede) [reply components/event-block/EventBlockRelatedStories.vue:10]: No, it's not in the design, but I'd like to give them the option for a future event.

**Pull request merged**